tool/gravity/cli/app.go

```diff
@@ -124,20 +124,31 @@ func listApps(env *localenv.LocalEnvironment, repository, appType string, showHi
 }
 
 // uninstallApp uninstalls gravity application from cluster
-func uninstallApp(env *localenv.LocalEnvironment, appPackage loc.Locator, portalURL string) error {
-	registryURL, err := localAppEnviron()
+func uninstallApp(env *localenv.LocalEnvironment, appPackage *loc.Locator) error {
+	apps, err := env.SiteApps()
 	if err != nil {
 		return trace.Wrap(err)
 	}
-	apps, err := env.AppServiceLocal(localenv.AppConfig{RegistryURL: registryURL})
-	if err != nil {
-		return trace.Wrap(err)
+
+	if appPackage == nil {
```

Comment:
Simplify 'app uninstall' to uninstall the cluster application if no package has been specified.